### DISCUSSION TITLE
[FEATURE REQUEST] Add support of cgroup_parent in docker_container

### DISCUSSION THREAD

**Discussion by mrsequeiraFF** [saltstack/salt#67500] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
I wanted to use the docker feature `cgroup_parent` but this isn't yet adopted by saltstack states.

**Describe the solution you'd like**
Add support of cgroup_parent parameter in this state: `salt.states.docker_container`.

**Describe alternatives you've considered**
I tried to pass the group_parent using the binds parameter but this doesn't work, as expected.
The only alternative is to run a cmd command

Please leave a thumbs up in case you have the same issue and give this ticket better visibility, thanks!

**Comment by welcome** [comment]
Hi there! Welcome to the Salt Community! Thank you for making your first contribution. We have a lengthy process for issues and PRs. Someone from the Core Team will follow up as soon as possible. In the meantime, here’s some information that may help as you continue your Salt journey.
Please be sure to review our [Code of Conduct](https://github.com/saltstack/salt/blob/master/CODE_OF_CONDUCT.md). Also, check out some of our community resources including:

  - [Community Wiki](https://github.com/saltstack/community/wiki)
  - [Salt’s Contributor Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html)
  - [Join our Community Slack](https://saltstackcommunity.herokuapp.com/)
  - [IRC on Freenode](https://webchat.freenode.net/#salt)
  - [Salt Project YouTube channel](https://www.youtube.com/channel/UCpveTIucFx9ljGelW63-BWg)
  - [Salt Project Twitch channel](https://www.twitch.tv/saltprojectoss)

There are lots of ways to get involved in our community. Every month, there are around a dozen opportunities to meet with other contributors and the Salt Core team and collaborate in real time. The best way to keep track is by subscribing to the Salt Community Events Calendar.
If you have additional questions, email us at saltproject@vmware.com. We’re glad you’ve joined our community and look forward to doing awesome things with you!

**Comment by sagetherage** [comment]
@mrsequeiraFF we will take a PR for review, thank you!
